frontend/src/App.js

```diff
@@ -97,14 +97,14 @@ import Footer from "./pages/components/footer";
 function App() {
 
   const data = useContext(UserContext);
-  console.log(data);
+  console.log(`data: ${data}`);
 
```

Comment:
What is this used for? Was this for testing?

Reply:
Yeah both lines 99, 100 were leftover code from a long time ago (before our work in albums) that was used for testing. Good catch, we should remove it.